README.md

```diff
@@ -46,21 +46,17 @@ optional arguments:
 
 #### Example: Tell me what snapshots *would* get deleted, but don't delete them (e.g. dryrun option).  Snapshots in us-east-1 with TagKey=MakeSnapshot, TagValue=DevTest14 which are older than 14 days (Policy is 14 Daily, 0 Weekly, 0 Monthly)
 ##### Note: This removes all Daily snapshots older than 14 days
-`$ python SnapClean.py -r us-east-1 -p 14:0:0 -k MakeSnapshot -v DevTest14 -a 123456789101 -d`
+`$ python3 SnapClean3.py -r us-east-1 -p 14:0:0 -k MakeSnapshot -v DevTest14 -a 123456789101 -d`
 
 #### Example: Delete Snapshots in us-east-1 with TagKey=MakeSnapshot, TagValue=DevTest14 which are older than 14 days (Policy is 14 Daily, 0 Weekly, 0 Monthly)
 ##### Note: This removes all Daily snapshots older than 14 days
-`$ python SnapClean.py -r us-east-1 -p 14:0:0 -k MakeSnapshot -v DevTest14 -a 123456789101`
-
-#### Example: Delete Snapshots in eu-west-1 with TagKey=MakeSnapshot, TagValue=Production which aren't part of the policy for 7 daily, 5 weekly, and 12 monthly (Policy is 7:5:12)
-##### Note: This removes all Daily snapshots older than 7 days, all Weekly older than 5 weeks, and all Monthly older than 12 months
-`$ python SnapClean.py -r eu-west-1 -p 7:5:12 -k MakeSnapshot -v Production -a 123456789101`
+`$ python3 SnapClean3.py -r us-east-1 -p 14:0:0 -k MakeSnapshot -v DevTest14 -a 123456789101`
 
 #### Example: Delete Snapshots in eu-west-1 with TagKey=MakeSnapshot, TagValue=True which are older than the following policy : 7 Daily, 5 Weekly, 12 Monthly).
 ##### Note: This removes all Daily snapshots older than 7 days, all Weekly older than 5 weeks, and all Monthly older than 12 months
-`$ python SnapClean.py -r us-east-1 -p 7:5:12 -k MakeSnapshot -v DevTest14 -a 123456789101`
+`$ python3 SnapClean3.py -r us-east-1 -p 14:0:0 -k MakeSnapshot -v DevTest14 -a 123456789101`
 
 #### Example: With DEBUG level logging, delete Snapshots in eu-west-1 with TagKey=MakeSnapshot, TagValue=True which are older than the following policy : 7 Daily, 5 Weekly, 12 Monthly).
 ##### Note: This removes all Daily snapshots older than 7 days, all Weekly older than 5 weeks, and all Monthly older than 12 months
-`$ python SnapClean.py -r us-east-1 -p 14:0:0 -k MakeSnapshot -v DevTest14 -a 123456789101 -l debug`
+`$ python3 SnapClean3.py -r us-east-1 -p 14:0:0 -k MakeSnapshot -v DevTest14 -a 123456789101 -l debug`
```

Comment:
This command does not align with the comment above it.  please correct the command (looks like it was broken even prior to this merge request.

Reply:
Fixed.